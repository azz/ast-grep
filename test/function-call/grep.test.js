import run from '..';

run(__dirname, 'baz ( );');
