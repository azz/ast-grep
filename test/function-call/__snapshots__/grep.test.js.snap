// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`function-call function-call.js: frames 1`] = `
  1 | foo(
  2 |   bar(
> 3 |     baz()
    |    ^^^^^
  4 |   )
  5 | );
  6 | 
`;

exports[`function-call function-call.js: raw 1`] = `

    baz()

`;
